src/main/scripts/org/reaktivity/specification/nukleus/http/streams/rfc7230/architecture/request.and.response/cooked/client.rpt

```diff
@@ -0,0 +1,36 @@
+#
+# Copyright 2016-2017 The Reaktivity Project
+#
+# The Reaktivity Project licenses this file to you under the Apache License,
+# version 2.0 (the "License"); you may not use this file except in compliance
+# with the License. You may obtain a copy of the License at:
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+
+# network <- http <- source (client)
+# Write to http/source, read from source/http
+
+property target "http"
+property newSourceOutputRef ${nuklei:newReferenceId()} # external scope
+
+connect "nukleus://streams/${target}"
+    option nukleus:source "source"
+    option nukleus:route ${newSourceOutputRef}
+    option nukleus:window 8192
+    option nukleus:transmission "duplex"
+connected
+
+write nukleus:extension [0x00] [0x07] ":scheme" [0x0x] "http"
+write nukleus:extension [0x00] [0x07] ":method" [0x03] "GET"
+write nukleus:extension [0x00] [0x05] ":path" [0x01] "/"
+write nukleus:extension [0x00] [0x0a] ":authority" [0x0e] "localhost:8080"
```

Comment:
might be helpful to support syntax like the following to avoid having to put in the string lengths:
```
write nukleus:extension [0x00] string ":scheme" string "http"
```